job/dhcp.go

```diff
@@ -141,13 +140,13 @@ func (j Job) setPXEFilename(rep *dhcp4.Packet, isPacket, isARM, isUEFI, isHTTPCl
 	var filename string
 	if !isPacket {
 		if j.PArch() == "hua" || j.PArch() == "2a2" {
-			filename = "ipxe/snp-hua.efi"
+			filename = "snp.efi"
 		} else if isARM {
-			filename = "ipxe/snp-nolacp.efi"
+			filename = "snp.efi"
 		} else if isUEFI {
-			filename = "ipxe/ipxe.efi"
+			filename = "ipxe.efi"
 		} else {
-			filename = "ipxe/undionly.kpxe"
+			filename = "undionly.kpxe"
```

Comment:
Was there a reason you removed the `/ipxe/` prefix from these files? I'm running into issues trying to deploy this in production where we're getting a 404 error when these files are requested from the root path. My understanding of the way things used to be is that auto.ipxe was served from the root, and everything else was from the `/ipxe/` subdirectory. I'm looking to possibly revert this behavior in a new PR but don't fully understand the context for the change.

Reply:
This prefix, `ipxe/` was removed here but added back in, for HTTP netboot clients only, here: https://github.com/tinkerbell/boots/blob/fdd7a4fb385659328bb8c3c6625741936c00aaa5/job/dhcp.go#L178

@nshalman changed this behavior with his last commit. `j.HTTPServerFQDN` to `conf.PublicFQDN`. This feels like why you are seeing issues.

Reply:
@ScottGarman and @nshalman, happy to jump on a call tomorrow if that would be helpful.